scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -4738,7 +4738,7 @@ function main {
                   println " >> ADVANCED >> CHAIN-ANALYSIS >> DOWNLOAD QUERIES"
                   println DEBUG "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
                   println DEBUG "\nDownloading official DBSync queries from Guild Operators GitHub site ..\n"
-                  if ! query_file_list=$(curl -s -m ${CURL_TIMEOUT} https://api.github.com/repos/cardano-community/guild-operators/contents/files/grest/queries?ref=${BRANCH}); then
+                  if ! query_file_list=$(curl -s -m ${CURL_TIMEOUT} https://api.github.com/repos/${G_ACCOUNT}/guild-operators/contents/files/grest/queries?ref=${BRANCH}); then
```

Comment:
ideally, any change to cntools.sh accompanies a version bump, but given that this one is 1-liner change where value resolved before and after are going to be the same, happy to go with current PR as is, will leave to @Scitz0 to merge if good

Reply:
Im fine to merge without patch version/changelog update. As noted, I dont think change neccessarily warrants a patch bump.